conf.d/spark.yaml.example

```diff
@@ -0,0 +1,20 @@
+init_config:
+
+instances:
+  #
+  # The Spark check retrieves metrics from YARN's ResourceManager. This
+  # check must be run from the Master Node and the ResourceManager URI must
+  # be specified below. The ResourceManager URI is composed of the 
+  # ResourceManager's hostname and port.
+  #
+  # The ResourceManager hostname can be found in the yarn-site.xml conf file
+  # under the property yarn.resourcemanager.address
+  #
+  # The ResourceManager port can be found in the yarn-site.xml conf file under
+  # the property yarn.resourcemanager.webapp.address
+  #
+  - resourcemanager_uri: http://localhost:8088
+    cluster_id: MySparkCluster
```

Comment:
Using `cluster_id` in the config could be misleading as users may think that they'd _have to_ set the `cluster_id` that's defined by Yarn. 

I realize we may have had a slight misunderstanding here, so to make things clearer, I think that we should:
- remove this `cluster_id` parameter in the yaml
- tag the metrics with the actual `cluster_id` as provided by Yarn if and only if the check is meant to be used along with another check that also tags by `cluster_id` (the Yarn check for instance). If it's not the case then we could: 1. have a parameter named `cluster_name` that would set a `cluster_name` tag and 2. add a comment in the yaml to explain that the parameter is mandatory but can be set to any user-defined value

What do you think?